docs/reference/logging/fluentd.md

```diff
@@ -0,0 +1,86 @@
+# Fluentd logging driver
+
+The `fluentd` logging driver sends container logs to the [Fluentd](http://www.fluentd.org/)
+as structured log data. Users can write these logs into any file
+systems or storages by using many output plugins.
+
+In addition to the text of the log message itself, the `fluentd` log
+driver sends the following metadata in structured log message:
+
+| Field            | Description |
+-------------------|-------------|
+| `container_id`   | The full 64-character container ID. |
+| `container_name` | The container name at the time it was started. If you use `docker rename` to rename a container, the new name is not reflected in the journal entries. |
+| `source`         | `stdout` or `stderr` |
+
+## Usage
+
+You can configure the default logging driver by passing the
+`--log-driver` option to the Docker daemon:
+
+    docker --log-driver=fluentd
+
+You can set the logging driver for a specific container by using the
+`--log-driver` option to `docker run`:
+
+    docker run --log-driver=fluentd ...
+
+## Options
+
+Users can specify some otions for Fluentd logging driver by using `--log-opt`
+option, like `--log-opt NAME=VALUE`.
+
+### fluentd-address
+
+Fluentd logging driver connects `localhost:24224` in default. Use
+`fluentd-address` option to specify another address to connect.
+
+    docker run --log-driver=fluentd --log-opt fluentd-address=myhost.local:24224 ...
+
+### fluentd-tag
+
+All Fluend's events have tag, which shows where these logs from. Fluentd
+logging driver specify it as `docker.CONTAINER_ID`.
+
+For value of this option, users can use some markup tag below:
```

Comment:
When specifying the value of this option, users can use the following markup tags: